ui/component/publish/post/postForm/view.jsx

```diff
@@ -0,0 +1,472 @@
+// @flow
```

Comment:
_Probably can do nothing about it for now._

Steps:
1. Edit an existing livestream claim.
2. Click on Post in the header -- the post editor is stuck in "loading..." until we clear Clear.
    - The "Save" button is grayed out, so it is probably good enough since this isn't a valid scenario + no harm can be done, but just highlighting in case we want to handle it.
    - The slightly concerning part just when Upload is clicked instead of Post.  We can still "Save" without a video and seems to be saving to the livestream.

Maybe Post  and Upload button needs to be grayed out, and the only way to access them again is for the user to clear the livestream edit first.  But not handling is probably ok for such a corner-case.

Reply:
Disabling the Post & Upload button would probably make things even more confusing because the cause might not be obvious for the user. Especially when the user has already left the livestream form and doesn't even know that there's a clear button. This issues should resolve itself once the states are separated.